### PR TITLE
Added Testcase to cover shift scenario.

### PR DESCRIPTION
**Pull request by amitsrivastava78**
This is one of the TODO items.

### CODE REVIEW

File: tensorflow/lite/kernels/kernel_util_test.cc [hunk 1]
```diff
@@ -251,6 +250,110 @@ TEST_F(KernelUtilTest, CheckAndPopulate) {
...
+  std::vector<int> per_channel_shift(3);
+
+  // Call and verify results for per channel case.
+  EXPECT_EQ(
+      kTfLiteOk,
+      PopulateConvolutionQuantizationParams(
+          &context, &input, &filter, &bias, &output, kTfLiteActRelu,
+          &multiplier, &shift, &output_activation_min, &output_activation_max,
+          per_channel_multiplier.data(), per_channel_shift.data()));
+  EXPECT_THAT(per_channel_multiplier, ::testing::ElementsAre(1073741824, 0, 0));
```
Comment (jianlijianli) [inline tensorflow/lite/kernels/kernel_util_test.cc]: Could you please add a comment why (1073741824, 0, 0) are expected for multiplier? Thanks.

**Review by jianlijianli — COMMENTED** [review]
Thanks for doing this. Add a small comment. Please take a look.

File: tensorflow/lite/kernels/kernel_util_test.cc (hunk 1 above)
Comment (amitsrivastava78) [reply tensorflow/lite/kernels/kernel_util_test.cc]: @jianlijianli , thanks for the review, i have updated as per your suggestion, kindly check and approve.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@jianlijianli , thanks for the review, i have updated as per your suggestion, kindly check and approve.

Regards
Amit

**Comment by jianlijianli** [comment]
Hi Amit, thanks for updating the comment. It seems per_channel_multiplier, per_channel_shift as well bias are vectors of size 3 but the scale is only a scalar; that doesn't seem to be the way PopulateConvolutionQuantizationParams is meant to be used. Just curious, what kind of use case it is testing? Thanks.

**Comment by amitsrivastava78** [comment]
@jianlijianli , thanks for pointing this out, i have updated the code again, 
This TC is basically to test the legacy quantization parameters, i mean the last if condition of the function PopulateConvolutionQuantizationParams . Hope this is ok.

Regards
Amit

**Review by jianlijianli — APPROVED** [review]
Thanks Amit for making the change.

**Pull request merged**